tests/benchdnn/graph/graph.cpp

```diff
@@ -560,20 +557,34 @@ int doit(const prb_t *prb, res_t *res) {
         if (res->state == SKIPPED) return OK;
 
         BENCHDNN_PRINT(3, "[INFO]: partition #%zd is unsupported!\n", i);
-        res->state = UNIMPLEMENTED;
-        return FAIL;
+        return res->state = UNIMPLEMENTED, OK;
     }
 
-    if (prb->expected_n_partition != 0
-            && partitions.size() != prb->expected_n_partition) {
+    if (partition_num_mismatch) {
         BENCHDNN_PRINT(0,
                 "Error: the expected number of partitions (%zu) doesn't "
                 "coincide with the actual number of partitions returned "
                 "(%zu).\n ",
                 prb->expected_n_partition, partitions.size());
-        return res->state = FAILED, FAIL;
+        SAFE(FAIL, WARN);
     }
+    return OK;
+}
+
+int doit(const prb_t *prb, res_t *res) {
+    if (bench_mode == bench_mode_t::list) return res->state = LISTED, OK;
+
+    skip_start(res);
+    if (res->state == SKIPPED) return OK;
+
+    const auto &dg = prb->dg;
+    const auto &graph_in_ports = dg.get_input_ports();
+    auto ograph = dg.to_graph(prb->fpmath_mode);
+    DNN_GRAPH_SAFE(ograph.finalize(), WARN, res);
 
+    const auto &partitions = ograph.get_partitions();
+    std::vector<size_t> end_opid_v {};
```

Comment:
SHould be gone, but if not, zero init is not needed.
```suggestion
    std::vector<size_t> end_opid_v;
```